### ISSUE TITLE
Powering the ESP32?

### ISSUE THREAD

**Issue by kpeatt** [wanaylor/esphome-autovorm#1]
Love this project! I was wondering if you'd done any experimentation to see what the voltage is on any of the lines and whether it would be possible to tap into them to power the ESP32 directly from the Moccamaster.

**Comment by wanaylor** [comment]
Yes I have actually; I did this in an early version using a Raspberry Pi Pico W but this should also work for the ESP. Here are the transformers I bought off Amazon. The attached picture shows two easily accessible points that are always at 120V so you should be able to wire up the transformer to those points and then feed the ESP from the low side of the transformer. I never got around to doing this because I can't get a reliable wifi connection with the ESP tucked into the Mochamaster so I have mine sticking out the back anyway and a cord to a wall wart isn't a huge issue for me.

Let me know if you get it to work, I've always wanted to get mine to work so it looks completely OEM.

https://www.amazon.com/dp/B07SGQ6XXR?ref=ppx_yo2ov_dt_b_fed_asin_title&th=1

<img width="842" alt="Mochamaster_Line" src="https://github.com/user-attachments/assets/74afded4-eed2-4e61-ba7e-8d297bf5e60c" />
